libs/wire-api/src/Wire/API/User.hs

```diff
@@ -872,35 +866,29 @@ modelChangePassword = Doc.defineModel "ChangePassword" $ do
   Doc.property "new_password" Doc.string' $
     Doc.description "New password (6 - 1024 characters)"
 
-instance ToJSON PasswordChange where
-  toJSON (PasswordChange old new) =
-    A.object
-      [ "old_password" A..= old,
-        "new_password" A..= new
-      ]
-
-instance FromJSON PasswordChange where
-  parseJSON = A.withObject "PasswordChange" $ \o ->
-    PasswordChange
-      <$> o A..:? "old_password"
-      <*> o A..: "new_password"
+instance ToSchema PasswordChange where
+  schema =
+    object "PasswordChange" $
+      PasswordChange
+        <$> cpOldPassword .= maybe_ (optField "old_password" schema)
```

Comment:
This should be
```haskell
cpOldPassword .= optField "old_password" (maybeWithDefault A.Null schema)
```

The difference is that now the field will be output even when its value is `Nothing`, and it will be given a value of `null`, whereas before it was omitted altogether. It probably makes no difference in practice, but to avoid any issues we should try to preserve the previous behaviour exactly if possible.